### PR TITLE
Add r-rnexml

### PR DESCRIPTION
**Pull request by nick-youngblut**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by nick-youngblut** [comment]
According to the contributing docs, I'm supposed to ask for a review of new packages, so @bioconda/core, can you please review! Please let me know if this isn't actually necessary, and I should just merge without a review.

File: recipes/r-rnexml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{% set version = '2.1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
```
Comment (daler) [inline recipes/r-rnexml/meta.yaml]: rm window-specific lines please

File: recipes/r-rnexml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+{% set native = 'm2w64-' if win else '' %}
```
Comment (daler) [inline recipes/r-rnexml/meta.yaml]: this one too

File: recipes/r-rnexml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - r-reshape2 >=1.2.2
+    - r-stringr >=1.0
+    - r-tidyr >=0.3.1
+    - r-uuid >=0.1_1
+    - r-xml2
+
+test:
+  commands:
+    - $R -e "library('RNeXML')"  # [not win]
+    - "\"%R%\" -e \"library('RNeXML')\""  # [win]
```
Comment (daler) [inline recipes/r-rnexml/meta.yaml]: only need 

```
- $R -e "library('RNeXML')"
```

Comment (nick-youngblut) [reply recipes/r-rnexml/meta.yaml]: Thanks for the clarification! I didn't realize that all windows lines should be removed. I've updated the recipe as you suggested.

**Comment by daler** [comment]
Thanks!

**Pull request merged**